README.md

```diff
@@ -178,6 +184,34 @@ viewed in markdown using the Rakefile in this directory.
 A web viewer is also available at [https://github.com/camptocamp/puppet-catalog-diff-viewer](https://github.com/camptocamp/puppet-catalog-diff-viewer)
 
 
+## See also
+
+### Upload facts to PuppetDB
+
+Standalone Ruby script `upload_facts.rb` that is used to upload yaml files with facts to
+PuppetDB. This is useful when you are upgradering Puppet version and uses two different
+Puppet Masters for this. Then you can use this script to upload facts from the old Master
+to the new one. The script can also be used to just refresh the facts in PuppetDB from
+the old Master. These facts are required to be able to compile the catalogs on the new
+Master.
+
+The script uses yaml-files in the same format as stored on the Puppet Master when real
+agents report their facts at the beginning of a Puppet Agent execution.
+
+The script is developed to be executed on the Puppet Master, so the yaml-facts files
+should be copied to the new Master using scp or similar, preferably to the
+`$(puppet config print vardir)/yaml/facts` directory.
+
+Then all files in the directory can be uploaded to PuppetDB by using this command:
+
+```shell
+$ ./upload_facts.rb $(puppet config print vardir)/yaml/facts/*.yaml
+```
+
+The script is available at [https://github.com/JohnEricson/upload_facts](https://github.com/JohnEricson/upload_facts).
+It's been verified to work with uploading facts from Puppet 3 to 5 Masters.
```

Comment:
`3 to 5` or `3 and 5`? Iirc the required API was removed in Puppet 4 and added again in 5.

Reply:
Yes, you are right! I'll suggest clarifying the text like this:
```suggestion
It's been verified to work with uploading facts from Puppet Masters running Puppet version 3 to Masters running version 5. It uses the [`/puppet/v3/facts/` API](https://puppet.com/docs/puppet/6.17/http_api/http_facts.html) which is available in version 3 and >= 5 of Puppet. This API was removed in Puppet 4.
```

Reply:
Alright, let's go with that then